docs/fr-FR/cli_configuration.md

```diff
@@ -0,0 +1,121 @@
+# Configuration de QMK CLI
+
+Ce document explique comment fonctionne la commande `qmk config`.
+
+# Introduction
+
+La configuration pour QMK CLI est un système clé/valeur. Chaque clé est composée d'une sous-commande et d'un argument séparé par une virgule. Cela permet une traduction simple et directe entre les clés de configuration et l'argument qu'elle définit.
+
+## Exemple simple
+
+Comme exemple, regardons la commande `qmk compile --keyboard clueboard/66/rev4 --keymap default`.
+
+Il y a deux arguments de ligne de commande qui peuvent être lu de la configuration:
+
+* `compile.keyboard`
+* `compile.keymap`
+
+Essayons de les définir:
+
+```shell
+$ qmk config compile.keyboard=clueboard/66/rev4 compile.keymap=default
+compile.keyboard: None -> clueboard/66/rev4
+compile.keymap: None -> default
+Ψ Wrote configuration to '/Users/example/Library/Application Support/qmk/qmk.ini'
+```
+
+Maintenant, je peux lancer la commande `qmk compile` sans avoir à spécifier mon clavier et keymap à chaque fois.
+
+## Définir les options par défaut
+
+Parfois, il est utile de partager une configuration entre plusieurs commandes. Par exemple, plusieurs commandes prennent un argument `--keyboard`. Plutôt que de devoir définir cette valeur pour chaque commande, vous pouvez définir une valeur d'utilisateur qui sera utilisée par toutes les commandes qui prennent cet argument.
+
+Exemple:
+
+```
+$ qmk config user.keyboard=clueboard/66/rev4 user.keymap=default
+user.keyboard: None -> clueboard/66/rev4
+user.keymap: None -> default
+Ψ Wrote configuration to '/Users/example/Library/Application Support/qmk/qmk.ini'
+```
+
+# CLI Documentation (`qmk config`)
+
+La commande `qmk config` est utilisée pour intéragir avec la configuration sous-jacente. Lancée sans argument, elle affiche la configuration courante. Lorsque des arguments sont définis, ils sont considérés comme étant des jetons de configuration, qui sont des chaînes de caractère ne contenant aucun espace avec le format suivant:
```

Comment:
The verbe `intéragir`doesn't take any accent, it should be `interagir`.

Reply:
Hey, merci pour les commentaires, mais ce pr a déjà été mergé. Tu pourrais ouvrir un pr pour les corrections. Sinon je vais essayer de le faire plus tard.

Reply:
Ok, je vais créer un PR.

Reply:
Ah et au fait, est ce que tu pourrais vérifier le pr https://github.com/qmk/qmk_firmware/issues/6995 ? Il nous manque un reviewer. Ce serait super utile, merci !

Reply:
Et voilà.